src/app/components/ATIAnalytics/atiUrl/index.ts

```diff
@@ -516,7 +516,17 @@ export const buildReverbPageSectionEventModel = ({
   advertiserID,
   url,
   experimentVariant,
+  componentSpecificTrackers,
+  blockSpecificTrackers,
 }: ATIEventTrackingProps): ReverbBeaconConfig => {
+  const itemCount = blockSpecificTrackers?.itemCount;
+  const blockLevelResourceId = blockSpecificTrackers?.resourceId;
+  const componentType = componentSpecificTrackers?.type;
+  const componentText = componentSpecificTrackers?.text;
+  const componentPosition = componentSpecificTrackers?.position;
+  const componentDuration = componentSpecificTrackers?.duration;
+  const componentResourceId = componentSpecificTrackers?.resourceId;
```

Comment:
Could we use destructuring here? 

Add all other fields here...
```suggestion
  const { type: componentType } = componentSpecificTrackers;
```